pyproject.toml

```diff
@@ -25,12 +25,10 @@ requires = [
 
     # now matches minimum supported version, keep here as separate requirement
     # to be able to sync more easily with oldest-supported-numpy
-    "numpy==1.19.5; python_version=='3.8' and platform_machine=='aarch64' and platform_python_implementation != 'PyPy'",
 
-    # arm64 on Darwin supports Python 3.8 and above requires numpy>=1.21.0
+    # arm64 on Darwin supports Python 3.9 and above requires numpy>=1.21.6
```

Comment:
The comment modification is incorrect, however this comment and the requirement below it can be deleted completely.